guides/partials.md

```diff
@@ -4,4 +4,33 @@ layout: guides
 permalink: /guides/partials/
 ---
 
-#Partials
+## Partials
+
+A Dust partial promotes "DRY" behavior for reusable/shared markup. A template is composed of one or more partials. Partials themselves can be composed of one or more partials. Partials rely on the JSON/context of the parent template invoking it. Lets peek under the covers to see how a Dust partial works.
```

Comment:
I don't know that there is a real distinction between a "template" and a "partial".